quiz/web/index.html

```diff
@@ -4,7 +4,8 @@
         <meta charset="UTF-8" />
         <link rel="icon" type="image/svg+xml" href="/vite.svg" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Vite + React + TS</title>
+        <link href="/dist/output.css" rel="stylesheet" />
```

Comment:
remove this line, as it's not really necessary.
Vite handles adding stylesheet on it's own.